hack/make-rules/build_with_container.sh

```diff
@@ -24,11 +24,20 @@ KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 MOUNTPATH="${MOUNTPATH:-/kubeedge}"
 KUBEEDGE_BUILD_IMAGE=${KUBEEDGE_BUILD_IMAGE:-"kubeedge/build-tools"}
 DOCKER_GID="${DOCKER_GID:-$(grep '^docker:' /etc/group | cut -f3 -d:)}"
+CONTAINER_RUN_OPTIONS="${CONTAINER_RUN_OPTIONS:--it}"
 
 echo "start building inside container"
-docker run --rm -u "${UID}:${DOCKER_GID}" \
+set +u
+# If in Github Action workflow or Travis CI, we disable -it option, or they will report error: the input device is not a TTY
+# In Github Action, environment variable GITHUB_ACTIONS is always set to true
+# In Travis CI, environment variable TRAVIS is always set to true
+if [[ $GITHUB_ACTIONS == true || $TRAVIS == true ]]; then
+  CONTAINER_RUN_OPTIONS=""
+fi
```

Comment:
How about set `CONTAINER_RUN_OPTIONS` to "" in github action and travis? So code can independent to the ci.